File: extensions/services/pg2luciddb/samples/php/dropschema.php
<?php

 // TODO: change your connection string here
 $connectionString = "host=localhost port=9999 dbname=LOCALDB user=sa password=";

 // open connection to the server
 $connection = pg_connect($connectionString) or die ("Unable to connect to PG2LucidDB: " . pg_last_error($connection)); 

 echo("Established connection to PG2LucidDB\n");

 // drop schema:
 pg_query("DROP SCHEMA PG2LUCIDDBTEST CASCADE");

 echo("PG2LUCIDDBTEST schema has been successfully deleted\n");

 // close connection:
 pg_close($connection);  

?>

File: extensions/services/pg2luciddb/samples/perl/dropschema.pl
#!perl

use DBI;
use DBD::Pg;

### change settings to your host with pg2luciddb:
my $dbname = "LOCALDB";
my $host = "localhost";
my $port = 9999;
my $dbuser = "sa";
my $dbpass = "";

$dbh = DBI->connect("dbi:Pg:dbname=$dbname;host=$host;port=$port", $dbuser, $dbpass, {AutoCommit => 0, RaiseError => 1, PrintError => 0});

print "Established connection to PG2LucidDB\n";

### drop schema:
$dbh->do("DROP SCHEMA PG2LUCIDDBTEST CASCADE");

print "PG2LUCIDDBTEST schema has been successfully deleted\n";

$dbh->disconnect();
